Hackney.Shared.Finance/Boundary/Request/PagingParametersRequest.cs

```diff
@@ -0,0 +1,13 @@
+﻿using Microsoft.AspNetCore.Mvc;
+
+namespace Hackney.Shared.Finance.Boundary.Request
+{
+    public abstract class PagingParametersRequest
```

Comment:
I know I was the one who specified this to be an `abstract` class, 
but I no longer thing it was the correct thing to do.

This class being `abstract` prevents it from being used as a standalone request object. There can easily be a case, where we have a 'Get Multiple Xs` endpoint that does not  have any filter parameters, but would benefit from paging.

If we either believe that it's possible that we'll run into such case, or we want to share this with the wider team, then it may be better to increase the flexibility of this package, by making this class a regular class.